mail/mail-server/Portfile

```diff
@@ -80,8 +90,9 @@ pre-build {
     set required_variants_message ""
 
     if { [catch {set result [registry_active postfix]}]
-        || [lindex [lindex ${result} 0] 3] \
-            ne "+[join ${postfix_required_variants} +]" } {
+         || ![lsearch_all \
+             [split [string trim [lindex [lindex ${result} 0] 3] +] +] \
+                 ${postfix_required_variants}] } {
```

Comment:
why are you not using `require_active_variants` here - you do include that PG but don't seem to make use of it? I am not sure re-inventing the wheel here beyond that PG is really required, but perhaps I am not understanding it from a quick look...

Reply:
Thanks for cleaning this up! Done.